scipy/sparse/dok.py

```diff
@@ -414,6 +411,9 @@ def __itruediv__(self, other):
         else:
             return NotImplemented
 
+    def __reduce__(self):
+        return dict.__reduce__(self)
```

Comment:
Ok, I can reproduce the failure on python 3. I would prefer implementing `__getstate__` and `__setstate__` (https://docs.python.org/3.1/library/pickle.html#pickle-state), though.